src/test/java/api/support/APITests.java

```diff
@@ -300,25 +310,63 @@ protected void useExampleFixedPolicyCirculationRules()
     MalformedURLException {
 
     log.info("Using fixed loan policy as fallback policy");
-    useLoanPolicyAsFallback(
+    useFallbackPolicies(
       loanPoliciesFixture.canCirculateFixed().getId(),
       requestPoliciesFixture.allowAllRequestPolicy().getId(),
-      noticePoliciesFixture.activeNotice().getId()
+      noticePoliciesFixture.activeNotice().getId(),
+      overdueFinePoliciesFixture.facultyStandard().getId()
     );
   }
 
-  protected void useLoanPolicyAsFallback(UUID loanPolicyId, UUID requestPolicyId,
-                                         UUID noticePolicyId)
+  protected void useFallbackPolicies(UUID loanPolicyId, UUID requestPolicyId,
+                                     UUID noticePolicyId, UUID overdueFinePolicyId)
     throws InterruptedException,
     ExecutionException,
     TimeoutException, MalformedURLException {
 
     circulationRulesFixture.updateCirculationRules(loanPolicyId, requestPolicyId,
-      noticePolicyId);
+      noticePolicyId, overdueFinePolicyId);
 
     warmUpApplyEndpoint();
   }
 
+
+  protected void setFallbackPolicies(LoanPolicyBuilder loanPolicyBuilder)
```

Comment:
Can you add javadoc that it uses the loan policy from the loanPolicyBuilder and the allowAllRequest policy, inactiveNotice policy and facultyStandard overdue fine policy?

Reply:
Done.